engine/src/main/java/org/hibernate/validator/constraints/br/CPF.java

```diff
@@ -43,16 +41,6 @@
 		@Pattern(regexp = "^(?:(?!888\\.?888\\.?888-?88).)*$"),
 		@Pattern(regexp = "^(?:(?!999\\.?999\\.?999-?99).)*$")
 })
-@Mod11Check.List({
-		@Mod11Check(checkDigitIndex = 12,
-				endIndex = 10,
-				treatCheck10As = '0',
-				ignoreNonDigitCharacters = true),
-		@Mod11Check(checkDigitIndex = 13,
-				endIndex = 12,
-				treatCheck10As = '0',
-				ignoreNonDigitCharacters = true)
```

Comment:
We had the idea of making `ignoreNonDigitCharacters` overridable via `@AnnotationOverrides`. Did this not work out?